LibM.Contracts/Author/AuthorViewDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibM.Contracts.Author
+{
+    public class AuthorViewDto
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+    }
```

Comment:
Yazarın kitap sayısıda eklense daha iyi olabilirdi :)

Reply:
güzel olurdu ama tabi işte talep etmedik. Belki bir sonraki sprintte böyle bir iş açabiliriz. :)

Reply:
Bir sonraki update'de bu konuyla alakalı çalışma yapılmak üzere not alıyorum.

Reply:
teşekkürler Berk Bey